Recording/Recording.s

```diff
@@ -11,6 +11,7 @@
 .set CMD_ITEM, 0x3B
 .set CMD_FRAME_BOOKEND, 0x3C
 .set CMD_GAME_END, 0x39
+.set CMD_MENU_FRAME, 0x3E
```

Comment:
Maybe put this under the command count and add comment that says these commands are not executed while in a recorded game